src/main/webapp/resources/js/types/ajax-response.d.ts

```diff
@@ -1,7 +1,12 @@
-interface AjaxErrorResponse {
-  error: string;
-}
+export = AJAXRESPONSE;
+export as namespace AJAXRESPONSE;
+
+declare namespace AJAXRESPONSE {
+  export interface AjaxErrorResponse {
+    error: string;
+  }
 
-interface AjaxSuccessResponse {
-  message: string;
+  export interface AjaxSuccessResponse {
```

Comment:
NIT: If you're not extending this can you just make it a type? You can take a look at https://youtu.be/zM9UPcIyyhQ

Reply:
Sure thing! Thanks for the video :)

Reply:
Updated to use type instead of interface in 04b8efed1da3f2b41b99f0a9a7edec22db2b6f06

Reply:
Thanks